fiftyone/operators/_types/pipeline.py

```
@@ -0,0 +1,119 @@
+        stages: a list of :class:`PipelineStage` instances
+    """
+
+    stages: list[PipelineStage] = dataclasses.field(default_factory=list)
+
+    def stage(
+        self,
+        operator_uri,
+        name=None,
+        num_distributed_tasks=None,
+        params=None,
+        **kwargs
```

Ruff (0.13.3)

Unused method argument: `kwargs`

(ARG002)